### PR TITLE
[jellyfin] add guide for jellyfin

### PR DESCRIPTION
**Pull request by knhash**
Jellyfin is the volunteer-built media solution that puts you in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.

### CODE REVIEW

**Comment by nichtmax** [comment]
This does not work for me: 

```
Unhandled Exception
System.UnauthorizedAccessException: Access to the path '/jellyfin' is denied.
 ---> System.IO.IOException: Permission denied
   --- End of inner exception stack trace ---
   at System.IO.FileSystem.CreateDirectory(String fullPath)
   at System.IO.Directory.CreateDirectory(String path)
   at Jellyfin.Server.Program.CreateApplicationPaths(StartupOptions options)
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
   at Jellyfin.Server.Program.<Main>(String[] args)

2021-11-19 09:21:33,545 DEBG 'jellyfin' stderr output:
Unhandled exception. System.UnauthorizedAccessException: Access to the path '/jellyfin' is denied.
 ---> System.IO.IOException: Permission denied
   --- End of inner exception stack trace ---
   at System.IO.FileSystem.CreateDirectory(String fullPath)
   at System.IO.Directory.CreateDirectory(String path)
   at Jellyfin.Server.Program.CreateApplicationPaths(StartupOptions options)
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
   at Jellyfin.Server.Program.<Main>(String[] args)
``` 

There are lots of instances where you use relative paths, e.g.

```
[isabell@stardust ~]$ mkdir jellyfin
```

please check out our [style guide](https://github.com/Uberspace/lab/blob/main/STYLE.md) and use only absolute paths:
> Always use full paths in commands. Don't assume the home directory or the html folder.

If you put the version into the folder you'll have to change it anytime you update in the supervisor deamon:

```
command=%(ENV_HOME)s/jellyfin/jellyfin_10.7.7/jellyfin
```

I'd suggest to symlink `~/jellyfin/jellyfin_10.7.7` to `~/jellyfin/current` and use this path in the config files, that makes updating (and rolling back) way easier.

**Review by nichtmax — CHANGES_REQUESTED** [review]
looks good, I left you some comments.

**Comment by knhash** [comment]
I started facing the same issue when I tried doing it on a new instance. Seemed to have something to do with where it was trying to access the logs or config while running in the background (directly running live seemed to work, via `supervisorctl` throws the above error.

Changed up the instructions slightly to accomodate it and tested on a new instance. This should work now.

File: source/guide_jellyfin.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+.. code-block:: bash
+
+  [isabell@stardust ~]$ wget -O ~/jellyfin/jellyfin.tar.gz https://repo.jellyfin.org/releases/server/linux/stable/combined/jellyfin_10.7.7_amd64.tar.gz
+  [isabell@stardust ~]$
+
+Extract the archive. Delete the archive post extraction.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ tar -xvzf ~/jellyfin/jellyfin.tar.gz  -C jellyfin
```
Comment (nichtmax) [inline source/guide_jellyfin.rst:66]: Please use absolute paths: `~/jellyfin`

File: source/guide_jellyfin.rst [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+.. code-block:: bash
+
+  [isabell@stardust ~]$ ln -s ~/jellyfin/jellyfin_10.7.7 ~/jellyfin/jellyfin
+  [isabell@stardust ~]$
+
+Create four sub-directories for Jellyfin data, and one to store your media library.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$  mkdir data cache config log library
```
Comment (nichtmax) [inline source/guide_jellyfin.rst:81]: Please use absolute paths. I think that should be `~/jellyfin/data` and so on?

File: source/guide_jellyfin.rst [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+.. code-block:: ini
+
+  [program:jellyfin]
+  command=bash %(ENV_HOME)s/jellyfin/jellyfin.sh
+
+.. include:: includes/supervisord.rst
+
+.. note::
+
+  If `jellyfin` is not ``RUNNING``, check your configuration and the logs using ``supervisorctl maintail``.
```
Comment (nichtmax) [inline source/guide_jellyfin.rst:130]: Still not working for me:

```
[moritz@stardust jellyfin]$ supervisorctl maintail
a child quit
2021-11-25 12:06:09,213 INFO spawned: 'jellyfin' with pid 12978
2021-11-25 12:06:09,220 DEBG 'jellyfin' stderr output:
/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory

2021-11-25 12:06:09,220 DEBG fd 9 closed, stopped monitoring <POutputDispatcher at 139918775711288 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stderr)>
2021-11-25 12:06:09,220 DEBG fd 7 closed, stopped monitoring <POutputDispatcher at 139918775710928 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stdout)>
2021-11-25 12:06:09,220 INFO exited: jellyfin (exit status 127; not expected)
2021-11-25 12:06:09,220 DEBG received SIGCLD indicating a child quit
2021-11-25 12:06:12,443 INFO spawned: 'jellyfin' with pid 13027
2021-11-25 12:06:12,449 DEBG 'jellyfin' stderr output:
/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory

2021-11-25 12:06:12,450 DEBG fd 9 closed, stopped monitoring <POutputDispatcher at 139918775447776 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stderr)>
2021-11-25 12:06:12,450 DEBG fd 7 closed, stopped monitoring <POutputDispatcher at 139918775682256 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stdout)>
2021-11-25 12:06:12,450 INFO exited: jellyfin (exit status 127; not expected)
2021-11-25 12:06:12,450 DEBG received SIGCLD indicating a child quit
2021-11-25 12:06:12,450 INFO gave up: jellyfin entered FATAL state, too many start retries too quickly
```

**Review by nichtmax — CHANGES_REQUESTED** [review]
I still can't get it working 😞

File: source/guide_jellyfin.rst (hunk 1 above)
Comment (knhash) [reply source/guide_jellyfin.rst]: Apologies, fixed now.

File: source/guide_jellyfin.rst (hunk 2 above)
Comment (knhash) [reply source/guide_jellyfin.rst]: Ack, fixed now.

File: source/guide_jellyfin.rst (hunk 3 above)
Comment (knhash) [reply source/guide_jellyfin.rst:130]: Ah, shell script needs to be changed to specify _your_ home directory. 
```/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory```
It is currently pointed to the default dummy user `isabell` in the shell script.

Comment (knhash) [reply source/guide_jellyfin.rst:130]: I had just highlighted the line to be updated as per the style guide. Should I explicitly call out the change while in there?

Comment (nichtmax) [reply source/guide_jellyfin.rst:130]: Just use `$USER`, no need to set your own username, that's unix built in ☺️

**Review by nichtmax — CHANGES_REQUESTED** [review]
We're close :D

File: source/guide_jellyfin.rst (hunk 3 above)
Comment (knhash) [reply source/guide_jellyfin.rst:129]: Aha! Fair.

**Comment by nichtmax** [comment]
Thanks! Good work! Now you earned your cup, get in touch via hallo@uberspace.de if you don't have the link I already sent you ☺️

**Pull request merged**